components/users/user.js

```diff
@@ -0,0 +1,32 @@
+import mongoose from 'mongoose'
+import bcryptjs from 'bcryptjs'
```

Comment:
Considering this application would be only in NodeJS, we can use `bcrypt` instead of `bcryptjs`. You can read more about it: https://stackoverflow.com/questions/54674387/how-do-nodes-bcrypt-and-bcryptjs-libraries-differ.

We can do it in another PR, but for you to know